{"uuid":"d1225286-885b-4a13-a3a4-d2f0c6ece0f7","historyId":"f0efcdca4a92817eec34bdb2bd775fbc","fullName":"TestCasesAndroid.ecommerceapp_testcase2.FillFormErrorScenario","labels":[{"name":"package","value":"TestCasesAndroid.ecommerceapp_testcase2"},{"name":"testClass","value":"TestCasesAndroid.ecommerceapp_testcase2"},{"name":"testMethod","value":"FillFormErrorScenario"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"AppiumTrainingHybridFramework"},{"name":"subSuite","value":"TestCasesAndroid.ecommerceapp_testcase2"},{"name":"host","value":"PPineda-EDP-100-958"},{"name":"thread","value":"8608@PPineda-EDP-100-958.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"Negative Testing - FillForm/Landing Page "},{"name":"story","value":"Story: Negative Flow Scenrio In General Store App"},{"name":"epic","value":"test3"}],"links":[],"name":"FillFormErrorScenario","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Failed Of Asertion! expected [Please enter your nam] but found [Please enter your name]","trace":"java.lang.AssertionError: Failed Of Asertion! expected [Please enter your nam] but found [Please enter your name]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat TestCasesAndroid.ecommerceapp_testcase2.FillFormErrorScenario(ecommerceapp_testcase2.java:102)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Test Description: Blank Text Field Then Submit Form is Expected To Fail","steps":[],"attachments":[{"name":"Page screenshot on failure","source":"ab9ae9bf-d616-4164-8029-fd77647250f8-attachment","type":"image/png"}],"parameters":[],"start":1701318303767,"stop":1701318339959}